src/hssm/distribution_utils/dist.py

```diff
@@ -592,109 +510,87 @@ def create_extra_pps_coord(self):
         return np.arange(2)
 
 
-def make_blackbox_op(logp: Callable) -> Op:
-    """Wrap an arbitrary function in a pytensor Op.
+def make_likelihood_callable(
+    loglik: pytensor.graph.Op | Callable | PathLike | str,
+    loglik_kind: Literal["analytical", "approx_differentiable", "blackbox"],
+    backend: Literal["pytensor", "jax", "cython", "other"] | None,
```

Comment:
We might want this. It is possible to implement analytical likelihoods in cython right? Our likelihood kinds might need some rethinking, since they are not mutually exclusive...

Reply:
agree that we may want to change the naming conventions entirely eventually. 
For now, cython stuff falls under blackbox (since no gradients) and I am just questioning if it really falls under 'backend' category in any relevant sense. 
It's just some pyhton callable in the end.